python/cugraph/cugraph/tests/utils/test_dataset.py

```diff
@@ -128,45 +85,48 @@ def is_symmetric(dataset):
 # Tests
 
 
-# setting download_dir to None effectively re-initialized the default
-def test_env_var():
-    os.environ["RAPIDS_DATASET_ROOT_DIR"] = "custom_storage_location"
+def test_modified_env_var(tmp_path):
+    # changing env var to tmp_path's location
+    os.environ["RAPIDS_DATASET_ROOT_DIR"] = str(tmp_path)
+    # calling this with None will reset to whatever the env var is set to
     datasets.set_download_dir(None)
 
-    expected_path = Path("custom_storage_location").absolute()
-    assert datasets.get_download_dir() == expected_path
+    assert datasets.get_download_dir() == tmp_path
 
-    del os.environ["RAPIDS_DATASET_ROOT_DIR"]
+    # reset to default
+    os.environ["RAPIDS_DATASET_ROOT_DIR"] = str(Path.home()) + "/.cugraph/datasets"
+    datasets.set_download_dir(None)
```

Comment:
Do tests assume the default download dir is being set specifically to `~/.cugraph/datasets`?  If so, could you just call this instead?
```suggestion
    datasets.set_download_dir(str(Path.home()) + "/.cugraph/datasets")
```

Reply:
That would be the case _unless_ specified otherwise by `RAPIDS_DATASET_ROOT_DIR`. By calling it with `None`, we let `download_dir` go back to whatever was the default (could be something else that's set by the testing environment).